src/main/webapp/app/exercises/programming/shared/lifecycle/programming-exercise-lifecycle.component.html

```diff
@@ -174,4 +174,23 @@ <h6 jhiTranslate="artemisApp.assessment.assessment"></h6>
         ></label>
         <jhi-help-icon placement="right auto" [text]="'artemisApp.programmingExercise.timeline.releaseTestsWithExampleSolutionTooltip'"></jhi-help-icon>
     </div>
+
+    <div class="form-check" *ngIf="(isAthenaEnabled$ | async) && !isExamMode">
```

Comment:
Note for reviewers: This is the checkbox at the bottom here in the programming exercise settings:
![screenshot-2023-09-04_001886](https://github.com/ls1intum/Artemis/assets/9006596/564168c0-325b-4eb4-9696-722078b124d4)